src/test/components/SignupForm.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import SignupForm from '../../components/SignupForm';
+
+
+describe('<SignupForm Test Suite>', () => {
```

Comment:
#### Nitpick
- Is this all you have for signup form?

Reply:
Alright I have added more test cases